### PR TITLE
Step Functions updates

### PR DESCRIPTION
**Pull request by cswatt**
### Merge instructions

- [x] Please merge after reviewing

### CODE REVIEW

File: content/en/serverless/step_functions/installation.md [hunk 1]
```diff
@@ -14,10 +14,21 @@ further_reading:
...
 ### How it works
-Datadog AWS Step Functions Monitoring collects logs and integration metrics from the AWS integration and uses ingested logs from AWS Step Functions to generate enhanced metrics and traces for your Step Function executions.
+AWS Step Functions is a fully managed service, and the Datadog Agent cannot be directly installed on Step Functions. However, Datadog can monitor Step Functions through logs.
+
+You can send your Step Functions logs to Datadog by using one of the following methods:
+
+- Use the [Datadog Forwarder][6]. For instructions, see the [Setup](#setup) section on this page.
+- Use Amazon Firehose. For instructions, see [Send AWS service logs to the Datadog Amazon Data Firehose destination][7].
+
+Datadog uses these ingested logs to generate [enhanced metrics][8] for your Step Function executions.
```
Comment (nine5two7) [inline content/en/serverless/step_functions/installation.md:24]: @cswatt  Here is some context that might be helpful to you.
- Logs are utilized to generate enhanced metrics and traces. 
- Customers can activate enhanced metrics by adding the DD_ENHANCED_METRICS tag to each of their Step Functions. When only enhanced metrics are enabled, customers are only billed for Serverless Workload Monitoring.
- Alternatively, customers can enable serverless tracing, which automatically includes enhanced metrics. With serverless tracing, customers are charged for both the Serverless Workload Monitoring and Serverless APM ([link to the pricing page](https://www.datadoghq.com/pricing/?product=serverless-monitoring#products)), which is calculated based on the count of state transitions. 
- To enable serverless tracing, customers can add the DD_TRACE_ENABLED tag to each Step Function or enable tracing for all Step Functions at the DD Forwarder level (see [this PR](https://github.com/DataDog/datadog-serverless-functions/pull/831)).

File: content/en/serverless/step_functions/_index.md [hunk 2]
```diff
@@ -15,12 +15,14 @@ further_reading:
...
 
 {{< img src="serverless/step_functions/overview1.png" alt="An AWS Step Function oveview tab." style="width:100%;" >}}
 
 ### How it works
-Datadog AWS Step Functions Monitoring collects logs and integration metrics from the AWS integration and uses ingested logs from AWS Step Functions to generate enhanced metrics and traces for your Step Function executions.
+Datadog AWS Step Functions Monitoring makes use of Cloudwatch metrics from the [AWS Step Functions integration][2] and Cloudwatch logs sent through the Datadog Forwarder or Amazon Data Firehose. The latter provides tracing and [enhanced metrics][3].
+
+{{< img src="serverless/step_functions/how_it_works.png" alt="Diagram showing two components of Datadog AWS Step Function monitoring: Cloudwatch metrics sent through the AWS Step Functions integration, and logs, traces, and enhanced metrics sent through the Datadog Lambda Forwarder or Amazon Data Firehose." style="width:100%;" >}}
```
Comment (nine5two7) [inline content/en/serverless/step_functions/_index.md:25]: The workflow is:
1) Logs are sent through the Datadog Lambda Forwarder or Amazon Data Firehose to Datadog. Both Forwarder and Firehose are running in customer's environment.
2) In Datadog, our internal service converts logs to traces and enhanced metrics. 
I just synced with @jakegreenberg1, and he will update the "how_it_works".png file accordingly.

File: content/en/serverless/step_functions/installation.md [hunk 3]
```diff
@@ -123,12 +134,10 @@ For developers using [Serverless Framework][4] to deploy serverless applications
...
 
-4. Enable enhanced metrics on your Step Function by adding a `DD_ENHANCED_METRICS` tag. Set the value to `true`. 
-5. Enable tracing on your Step Function by adding a `DD_TRACE_ENABLED` tag. Set the value to `true`. This is only needed if `DD_STEP_FUNCTIONS_TRACE_ENABLED` hasn't been set by the Forwarder. 
-6. Set up tags. Open your AWS console and go to your Step Functions state machine. Open the *Tags* section and add `env:<ENV_NAME>` and `service:<SERVICE_NAME>` tags. The `env` tag is required to see traces in Datadog, and it defaults to `dev`. The `service` tag defaults to the state machine's name.
-7. For Node.js and Python runtimes, you can link your Step Function traces to Lambda traces. On the Lambda Task, set the `Parameters` key with the following: 
+4. Set up tags. Open your AWS console and go to your Step Functions state machine. Open the *Tags* section and add `env:<ENV_NAME>` and `service:<SERVICE_NAME>` tags. The `env` tag is required to see traces in Datadog, and it defaults to `dev`. The `service` tag defaults to the state machine's name.
```
Comment (nine5two7) [inline content/en/serverless/step_functions/installation.md:139]: Customers are also recommended to set the version tag for Step Functions. Without setting this tag, the default value is "1.0".

**Review by nine5two7 — APPROVED** [review]
LGTM

**Comment by cswatt** [comment]
Holding for new image.

**Review by rtrieu — APPROVED** [review]
LGTM

**Pull request merged**